build/makelib/golang.mk

```diff
@@ -177,6 +183,12 @@ $(GO_TOOLS_DIR)/glide:
 	@mv $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)/glide $(GO_TOOLS_DIR)/glide
 	@rm -r $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)
 
+.PHONY: $(GO_TOOLS_DIR)/golint
+$(GO_TOOLS_DIR)/golint:
+	@echo "installing golint"
+	@mkdir -p $(GO_TOOLS_DIR)
+	@GOBIN=`pwd`/tools $(GOHOST) get github.com/golang/lint/golint
```

Comment:
i'm not great with make syntax.  does this overwrite the value of the `GOBIN` variable?  does this result in the compiled golint binary existing in the tools dir?